src/main/java/com/google/sps/classes/Utility.java

```diff
@@ -115,6 +114,39 @@ public static int getUserId() {
 
     return userId;
   }
+
+  /**
+   * Returns username of a user given their ID.
+   * Returns empty string if user was not found.
+   */
+  public static String getUsername(int userId) {
+    String username = "";
+
+    // Set up query to get username.
+    String query = "SELECT * FROM User WHERE id = " + userId;
+
```

Comment:
It would probably be better if you just queried the username instead of all the columns.
```suggestion
    // Set up query to get username.
    String query = "SELECT username FROM User WHERE id = " + userId;

```

Reply:
Done!